src/main/webapp/app/vendor.ts

```diff
@@ -5,3 +5,252 @@ import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/observable/of';
 import 'rxjs/add/observable/throw';
+import { library } from '@fortawesome/fontawesome-svg-core';
```

Comment:
I think a merge conflict occurred here. As far as I know, @thilo-behnke moved this into its own module